frappe/core/doctype/doctype/doctype.js

```diff
@@ -33,10 +33,17 @@ frappe.ui.form.on('DocType', {
 			}
 		}
 
+		const customize_form_link = "<a href='/app/customize-form'>Customize Form</a>"
 		if(!frappe.boot.developer_mode && !frm.doc.custom) {
 			// make the document read-only
 			frm.set_read_only();
-		}
+			frm.dashboard.add_comment(__("DocTypes can not be modified, please use {0} instead", [customize_form_link]), "blue", true);
+		} else if (frappe.boot.developer_mode) {
+			let msg = __("This site is running in develop mode. Any change made here will be updated in code.");
```

Comment:
```suggestion
			let msg = __("This site is running in develop mode. Any change made here will be updated in code, and may leave other sites on your bench in an inconsistent state.");
```

Is the truth too wordy? I'm not sure...take it or leave it.

Reply:
Users don't really know or understand multi-tenancy, should be bother adding this? 🤔